googlepay/src/main/java/com/adyen/checkout/googlepay/model/GooglePayParamUtils.kt

```diff
@@ -0,0 +1,15 @@
+package com.adyen.checkout.googlepay.model
+
+import com.adyen.checkout.core.exception.CheckoutException
+import com.adyen.checkout.googlepay.util.AllowedCardNetworks
+
+internal object GooglePayParamUtils {
+
+    fun mapTxVariantToGooglePayCode(txVariant: String): String {
```

Comment:
I'm not sure if txVariant is the right name here, maybe brand is better.

Reply:
✔️